src/System.Memory/ref/System.Memory.cs

```diff
@@ -87,6 +87,7 @@ public static class SpanExtensions
     {
          public static int IndexOf<T>(this Span<T> span, T value) where T:struct, IEquatable<T> { throw null; }
```

Comment:
If this is just a temporary hack until the Span<T> codegen is fixed, you should keep it in corefxlab.